### DISCUSSION TITLE
7.8.3 专业版，window 11闪退

### DISCUSSION THREAD

**Discussion by konaisland** [getlantern/lantern-forum-cn#319] [General]
闪退，使用删除global.yaml文件可以启动，但是还要从新认证专业版，端口也变了，根本不好用，请赶紧解决。

**Comment by MaxMKMax** [comment]
@konaisland 

您好。对于造成的不便，我们深表歉意。好的，我会帮您转达给我们的负责人。抱歉，周末团队处于离线状态。他们一上线回来就能修复好. 请您耐心等待. 

非常抱歉影响了您的体验感. 请您再给我们一次机会. 谢谢您!

**Comment by MaxMKMax** [comment]
@konaisland 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)


当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。
